pkg/cdi/cache_test.go

```diff
@@ -812,6 +814,102 @@ devices:
 	}
 }
 
+func TestTooManyOpenFiles(t *testing.T) {
+	if runtime.GOOS != "linux" {
```

Comment:
Would it make sense to move this test to the `cache_test_linux.go` ? Would it work on other *nixes or Darwin?

Reply:
It could be made to work on other un*xen as well. Would need an OS-specific implementation (and a renaming) of getProcStatusFdSize. But it probably does not make much sense at the moment so we might as well move all the related code to cache_test_linux.go.

Reply:
Or cache_linux_test.go...

Reply:
Moved to a linux-specific test file.